man/import.q.feedback.Rd

```diff
@@ -13,13 +18,13 @@
 		A relative path to a directory structure where:
 		\itemize{
 			\item folders are \code{conditions} (such as before or after), if there are more than one conditions.
-			\item filenames of \code{*.csv}s are participant names (might be given pseudneums)
-			\item csvs containing raw feedback, beginning with an arbitary header line (ignored), and the following columns, starting from the left:
+			\item filenames of .CSV are participant names (might be given pseudneums)
+			\item .CSV containing raw feedback, beginning with an arbitary header line (ignored), and the following columns, starting from the left:
 			\enumerate{
-				\item An automatic hash from \code{\link{make.cards}} or a \code{manual.lookup} id.
-				Each id only occurs once.
+				\item An automatic hash from \code{\link{make.cards}} or a \code{manual.lookup} ID.
+				Each ID only occurs once.
 				\item The full feedback.
-				\item Logical indicator whether current line should be ignored (TRUE).
+				\item Logical indicator whether current line should be ignored (in which case it should be set to \code{TRUE}).
```

Comment:
`\item Logical value indicating whether current line should be ignored (in which case it should be set to \code{TRUE}).`


(?) if this third column is not included, would the function work either way? (I'd assume that rarely I would like to ignore any of the feedback)